support-config/src/main/scala/com/gu/support/config/ZuoraConfigProvider.scala

```diff
@@ -2,6 +2,7 @@ package com.gu.support.config
 
 
 import com.gu.support.catalog.{ProductRatePlanChargeId, ProductRatePlanId}
+import com.gu.support.workers.BillingPeriod.Annual
```

Comment:
I moved the objects inside BillingPeriod as `com.gu.support.workers.Monthly` didn't make so much sense, noticed  when I was writing the tests

Reply:
I do think there is value in refactoring but at the same time this PR has 50 files changed for what is actually a very small piece of functionality. Also quite often refactoring can be more about changing code to fit our own personal preferences for code structure rather than any actual tangible benefit. 
I would rather assume that the person who originally wrote the code had a reason for doing it that way and work with the existing structure, hopefully resulting in a more consistent codebase.

Reply:
true, that certainly fits in with chestertons fence https://fs.blog/2020/03/chestertons-fence/
i.e. don't change something until you understand why it was done that way.

I did just assume that it was an oversight to have multiple "Monthly" classes in support-models with no positive distinction - here actually the second one is monthly Billing Period (this screenshot is master)
![image](https://user-images.githubusercontent.com/7304387/84412226-d6b21e00-ac06-11ea-8474-51cd5a42bdc6.png)

However I will remove this as I see that it makes it hard to review this code even if we do want to change it.